worlds/mmx3/Regions.py

```diff
@@ -0,0 +1,514 @@
+    connect(world, RegionName.neon_tiger_hill, RegionName.neon_tiger_boss)
+
+    # Connect Gravity Beetle
+    connect(world, RegionName.gravity_beetle, RegionName.gravity_beetle_start)
+    connect(world, RegionName.gravity_beetle_start, RegionName.gravity_beetle_bit_byte)
+    connect(world, RegionName.gravity_beetle_bit_byte, RegionName.bit_byte)
+    connect(world, RegionName.gravity_beetle_bit_byte, RegionName.gravity_beetle_outside)
+    connect(world, RegionName.gravity_beetle_outside, RegionName.gravity_beetle_inside)
+    connect(world, RegionName.gravity_beetle_inside, RegionName.gravity_beetle_boss)
+
+    # Connect Blast Hornet
+    connect(world, RegionName.blast_hornet, RegionName.blast_hornet_start)
+    connect(world, RegionName.blast_hornet_start, RegionName.blast_hornet_conveyors)
+    connect(world, RegionName.blast_hornet_conveyors, RegionName.blast_hornet_shurikein)
+    connect(world, RegionName.blast_hornet_shurikein, RegionName.blast_hornet_outside)
+    connect(world, RegionName.blast_hornet_outside, RegionName.blast_hornet_bit_byte)
+    connect(world, RegionName.blast_hornet_bit_byte, RegionName.bit_byte)
+    connect(world, RegionName.blast_hornet_bit_byte, RegionName.blast_hornet_boss)
+    
+    # Connect Vile
+    connect(world, RegionName.blizzard_buffalo_start, RegionName.vile)
+    connect(world, RegionName.volt_catfish_start, RegionName.vile)
+    connect(world, RegionName.crush_crawfish_start, RegionName.vile)
+    connect(world, RegionName.vile, RegionName.vile_before)
+    connect(world, RegionName.vile_before, RegionName.vile_boss)
+    connect(world, RegionName.vile_boss, RegionName.vile_after)
+
+
+    # Connect Dr. Doppler Lab
+    connect(world, RegionName.intro_stage, RegionName.dr_doppler_lab)
+
+    # Connect Dr. Doppler Lab levels
+    if world.options.doppler_all_labs.value:
+        connect(world, RegionName.dr_doppler_lab, RegionName.dr_doppler_lab_1)
+        connect(world, RegionName.dr_doppler_lab, RegionName.dr_doppler_lab_2)
+        connect(world, RegionName.dr_doppler_lab, RegionName.dr_doppler_lab_3)
+        connect(world, RegionName.dr_doppler_lab, RegionName.dr_doppler_lab_4)
+    else:
+        connect(world, RegionName.dr_doppler_lab, RegionName.dr_doppler_lab_1)
+        connect(world, RegionName.dr_doppler_lab_1, RegionName.dr_doppler_lab_2)
+        connect(world, RegionName.dr_doppler_lab_2, RegionName.dr_doppler_lab_3)
+        connect(world, RegionName.dr_doppler_lab_3_boss, RegionName.dr_doppler_lab_4)
+    
+    connect(world, RegionName.dr_doppler_lab_3, RegionName.dr_doppler_lab_3_rematches)
+    connect(world, RegionName.dr_doppler_lab_3_rematches, RegionName.dr_doppler_lab_3_after_rematches)
+    connect(world, RegionName.dr_doppler_lab_3_after_rematches, RegionName.dr_doppler_lab_3_boss)
+    
+    connect(world, RegionName.dr_doppler_lab_4, RegionName.dr_doppler_lab_4_boss)
+
+    # Connect checkpoints
+    if world.options.logic_helmet_checkpoints.value:
```

Comment:
```suggestion
    if world.options.logic_helmet_checkpoints:
```
No need for .value here